ice/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 * Increased min verison of `log` dependency to `0.4.16`. [#250 Fix log at ^0.4.16 to make tests compile](https://github.com/webrtc-rs/webrtc/pull/250) by [@k0nserv](https://github.com/k0nserv).
 * Incresed serde's minimum version to 1.0.102 [#243 Fixes for cargo minimal-versions](https://github.com/webrtc-rs/webrtc/pull/243) contributed by [algesten](https://github.com/algesten)
+* Promote agent lock in ice_gather.rs create_agent() to top level of the function to avoid a race condition. [#290 Promote create_agent lock to top of function, to avoid race condition](https://github.com/webrtc-rs/webrtc/pull/290) contributed by [efer-ms](https://github.com/efer-ms)
```

Comment:
Move this up so it's under "unreleased" not "v.0.8.0" please :)

Reply:
Gah. Sorry. Should it be "## v0.8.0" instead of ###? I think that's what threw me. Looked like 0.8.0 was under unreleased.